io.github.aganzha.Stage.json

```diff
@@ -0,0 +1,60 @@
+{
+   "app-id": "io.github.aganzha.Stage",
+   "runtime": "org.gnome.Platform",
+   "runtime-version": "46",
+   "sdk": "org.gnome.Sdk",
+   "sdk-extensions": ["org.freedesktop.Sdk.Extension.rust-stable"],
+   "command": "stage",
+   "finish-args": [
+      "--filesystem=home",
+      "--share=ipc",
+      "--share=network",
+      "--socket=fallback-x11",
+      "--socket=wayland",
+      "--socket=ssh-auth",
+      "--socket=gpg-agent",
+      "--device=dri"
+   ],
+   "build-options": {
+      "append-path": "/usr/lib/sdk/rust-stable/bin"
+   },
+   "modules": [{
+      "name": "stage",
+      "buildsystem": "simple",
+      "build-options": {
+         "build-args": [
+            "--share=network"
+         ],
+         "env": {
+            "CARGO_HOME": "/run/build/stage/cargo"
+         }
+      },
+      "build-commands": [
+         "cargo fetch --verbose",
+         "cargo build --release --verbose",
+         "glib-compile-schemas ./src && cp ./src/gschemas.compiled ./target/release",
+         "install -Dm755 ./target/release/stage -t /app/bin/",
+         "install -Dm744 ./target/release/gschemas.compiled -t /app/bin/",
+         "install -Dm644 io.github.aganzha.Stage.metainfo.xml -t /app/share/metainfo",
+         "install -Dm644 io.github.aganzha.Stage.desktop -t /app/share/applications",
+         "install -Dm644 ./icons/16x16/io.github.aganzha.Stage.png -t /app/share/icons/hicolor/16x16/apps",
+         "install -Dm644 ./icons/32x32/io.github.aganzha.Stage.png -t /app/share/icons/hicolor/32x32/apps",
+         "install -Dm644 ./icons/48x48/io.github.aganzha.Stage.png -t /app/share/icons/hicolor/48x48/apps",
+         "install -Dm644 ./icons/64x64/io.github.aganzha.Stage.png -t /app/share/icons/hicolor/64x64/apps",
+         "install -Dm644 ./icons/128x128/io.github.aganzha.Stage.png -t /app/share/icons/hicolor/128x128/apps",
+         "install -Dm644 ./icons/256x256/io.github.aganzha.Stage.png -t /app/share/icons/hicolor/256x256/apps",
```

Comment:
this should really be done by a proper buildsystem

Reply:
Hello. What do you mean? Must I remove those lines from manifest and replace them by some command?